README.md

```diff
@@ -50,6 +50,16 @@ For an easy and fast deployment of your application we recommend [heroku](https:
 
 <a href="https://heroku.com/deploy?template=https://github.com/sphereio/sphere-sunrise"><img src="https://www.herokucdn.com/deploy/button.png" alt="Deploy"></a>
 
+## Docker
+
+[![Docker build](http://dockeri.co/image/sphereio/sunrise)](https://registry.hub.docker.com/u/sphereio/sunrise/)
+
+You can also use a ready-to-run docker container.
+
+```bash
+$ docker run -it --rm -p 9000:9000 sphereio/sunrise
```

Comment:
Yes, have a look at the README regarding integration tests. There you see the following list of environment variables:

```bash
export SPHERE_SUNRISE_IT_PROJECT="your-project-key"
export SPHERE_SUNRISE_IT_CLIENT_SECRET="your-the-client-secret"
export SPHERE_SUNRISE_IT_CLIENT_ID="your-client-id"
```
which you can provide via the `-e` docker parameter.